src/rpc/misc.cpp

```diff
@@ -63,6 +63,7 @@ UniValue getinfo(const UniValue& params, bool fHelp)
             "  \"unlocked_until\": ttt,      (numeric) the timestamp in seconds since epoch (midnight Jan 1 1970 GMT) that the wallet is unlocked for transfers, or 0 if the wallet is locked\n"
             "  \"paytxfee\": x.xxxx,         (numeric) the transaction fee set in " + CURRENCY_UNIT + "/kB\n"
             "  \"relayfee\": x.xxxx,         (numeric) minimum relay fee for non-free transactions in " + CURRENCY_UNIT + "/kB\n"
+            "  \"sizelimit\" : n,            (numeric) BIP100 block size limit as of the last block\n"
```

Comment:
I know, but shouldn't there be some explanatory text, such as...

- BIP100 block size limit as of the last block\n"
+BIP100 block size limit (in Bytes) as of the last block\n"